lib/constellation.rb

```diff
@@ -0,0 +1,39 @@
+class ConstellationHttpException < StandardError
+end
```

Comment:
Only if you had wanted to render the same type of JSON response as your doing for `ConstellationHttpException`?